include/aspect/material_model/steinberger.h

```diff
@@ -183,6 +183,14 @@ namespace aspect
                                    const std::vector<double> &compositional_fields,
                                    const Point<dim> &position) const;
 
+        /**
+        * Returns the volume fractions of all the phases over all of the current
+        * evaluation points. The outer loop is over the phases, and the inner
+        * loop is over all the evaluation points.
```

Comment:
You still need to update the documentation. (the function doesn't return the volume fractions any more). 
I would also phrase the second part in a different way: You want to say what this new input object is and how it is filled. Something like: 
This function uses the MaterialModelInputs &in to fill the output_values of the phase_volume_fractions_out output object with [...]. This output_values object is a vector of vectors; the outer vector is expected to have a size that equals the number of unique phases, the inner vector is expected to have a size that equals the number of evaluation points.